frappe/utils/safe_exec.py

```diff
@@ -60,6 +61,16 @@ def check_name(self, node, name, *args, **kwargs):
 		return super().check_name(node, name, *args, **kwargs)
 
 
+class FrappePrintCollector(PrintCollector):
+	"""Collect written text, and return it when called."""
+
+	def _call_print(self, *objects, **kwargs):
+		output = io.StringIO()
+		print(*objects, file=output, **kwargs)
+		frappe.log(output.getvalue().strip())
```

Comment:
This doesn't work without enabling debug logging using `logging` key in site config. 

Can we ignore it and do it always in case of safe exec prints? :thinking: